models/mail.go

```diff
@@ -157,10 +157,13 @@ func composeTplData(subject, body, link string) map[string]interface{} {
 
 func composeIssueCommentMessage(issue *Issue, doer *User, content string, comment *Comment, tplName base.TplName, tos []string, info string) *mailer.Message {
 	subject := issue.mailSubject()
-	issue.LoadRepo()
+	err := issue.LoadRepo()
+	if err != nil {
+		log.Error("LoadRepo: %v", err)
```

Comment:
This function should return error as second return parameter I think.

Reply:
The problem is it is called in `SendAsync` which just puts the mail in a queue to be sent. There's not much room for error handling there.

Reply:
Or we have to return nil here, otherwise it may panic when invoke `issue.Repo.HTMLURL()`.